docs/cadc/cadc.rst

```diff
@@ -27,7 +26,10 @@ data.
 
   >>> from astroquery.cadc import Cadc
   >>>
-  >>> warnings.filterwarnings('ignore')  # turn off astropy warnings (optional)
+  >>> # turn off astropy and astroquery warnings (optional)
+  >>> import warnings
+  >>> warnings.filterwarnings('ignore', module='astropy.io.votable')
```

Comment:
I'm not sure how I feel about this at the very top of the docs for users. Are there disturbingly many warnings coming from there?

Reply:
There can be, in some cases.  Maybe this is actually worth putting in the code, wrapping the VOTable parser?  If we know CADC will definitely return VOTables that have warnings, we can just suppress those warnings and assume the parsing will work.

Reply:
Yes, I would prefer to do that. Ignore the warning that we know about and know we don't need to care, but teaching the users from the beginning to ignore them doesn't seem to be the best approach.

Reply:
These are the warnings:
```
WARNING: W47: None:68:6: W47: Missing arraysize indicates length 1 [astropy.io.votable.converters]
WARNING: W50: None:146:6: W50: Invalid unit string 'per sq. deg' [astropy.io.votable.tree]
WARNING: W32: None:251:6: W32: Duplicate ID 'metaChecksum' renamed to 'metaChecksum_2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W32: None:254:6: W32: Duplicate ID 'accMetaChecksum' renamed to 'accMetaChecksum_2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W32: None:257:6: W32: Duplicate ID 'obsID' renamed to 'obsID_2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W32: None:266:6: W32: Duplicate ID 'metaRelease' renamed to 'metaRelease_2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W32: None:320:6: W32: Duplicate ID 'lastModified' renamed to 'lastModified_2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W33: None:251:6: W33: Column name 'metaChecksum' renamed to 'metaChecksum 2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W33: None:254:6: W33: Column name 'accMetaChecksum' renamed to 'accMetaChecksum 2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W33: None:257:6: W33: Column name 'obsID' renamed to 'obsID 2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W33: None:266:6: W33: Column name 'metaRelease' renamed to 'metaRelease 2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W33: None:320:6: W33: Column name 'lastModified' renamed to 'lastModified 2' to ensure uniqueness [astropy.io.votable.tree]
WARNING: W48: None:873:10: W48: Unknown attribute 'value' on OPTION [astropy.io.votable.tree]
WARNING: W48: None:874:10: W48: Unknown attribute 'value' on OPTION [astropy.io.votable.tree]
WARNING: W48: None:887:10: W48: Unknown attribute 'value' on OPTION [astropy.io.votable.tree]
WARNING: W48: None:888:10: W48: Unknown attribute 'value' on OPTION [astropy.io.votable.tree]
```
VOTable is the underlying protocol and I doubt users know what to do about them. They look ugly in a documentation code. Maybe I should look into fixing them...

Reply:
Fixing them is the best, but second best is what @keflavich suggested, catch them in the code and never show to the users.

Reply:
Issues have been addressed with a coupled of noted exceptions. Very good comments - thanks.
Next, I'm going to boost up the coverage for the cadc.core module.